src/plugins/intel_cpu/src/nodes/topk.cpp

```diff
@@ -1782,7 +1783,23 @@ struct jit_uni_topk_kernel_f32 : public jit_uni_topk_kernel, public jit_generato
     }
 
     void bubble_swap_xmm(Xmm xmm_val_a, Xmm xmm_idx_a, Xmm xmm_val_b, Xmm xmm_idx_b, bool cmp_val = true) {
+        compare_node_xmm(xmm_val_a, xmm_idx_a, xmm_val_b, xmm_idx_b, xmm_tmp, _cmp_eq_oq, _cmp_nle_us, false);
```

Comment:
Shall we make this part conditionally? This introduces additional overhead no matter if it is required to be stable or not